cpp/tests/CMakeLists.txt

```diff
@@ -333,6 +332,12 @@ ConfigureTest(
   GPUS 1
   PERCENT 30
 )
+ConfigureTest(
+  HYBRID_SCAN_TEST io/experimental/hybrid_scan_common.cpp
+  io/experimental/hybrid_scan_filters_test.cpp io/parquet_common.cpp io/parquet_test.cpp
+  GPUS 1
+  PERCENT 30
```

Comment:
Is this just copy-paste or does this test actually use a significant amount of GPU memory (>1 GB)?